tests/SmokeTests.js

```diff
@@ -6,14 +6,74 @@ describe('Smoke Testing', () => {
         App.openHomePage()
         HomePage.ToolsQaImgIsVisible()
     });
-    it('Check if Elements entity can be viewed on Home Page', async () => {
+    it.skip('Check if all the six entities can be viewed on Home Page', async () => {
         HomePage.ElementsEntityIsVisible()
-        comst h5=HomePage.h5
-        await expect(h5).toHaveText('Elements')
+        const h5_1 = await HomePage.h5_Elements
+        await expect(h5_1).toHaveText('Elements')
+
+        HomePage.FormsEntityIsVisible()
```

Comment:
This line is returning you a promise instead returning you the results of the `FormsEntityIsVisible()` function. To get the results use `await` keyword.

Reply:
If I use await before HomePage.FormsEntityisVisible and hover with mouse over it I get this message: "await has no effects on the type of this expression"